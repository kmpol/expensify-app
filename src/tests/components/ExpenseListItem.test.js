import React from 'react'
import { shallow, configure } from 'enzyme'
import Adapter from 'enzyme-adapter-react-16';
import toJSON from 'enzyme-to-json'

import expenses from '../fixtures/expenses'
import ExpenseListItem from '../../components/ExpenseListItem'

configure({ adapter: new Adapter() })

test('Should render ExpenseListItem with one item from fixtured expense', () => {
    const wrapper = shallow(<ExpenseListItem {...expenses[0]} />)
    expect(toJSON(wrapper)).toMatchSnapshot()
})