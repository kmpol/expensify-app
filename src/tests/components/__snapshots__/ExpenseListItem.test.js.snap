// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseListItem with one item from fixtured expense 1`] = `
<div>
  <Link
    to="/edit/1"
  >
    <h3>
      Gum
    </h3>
  </Link>
  <p>
    123
     - 
    0
  </p>
</div>
`;
